tests/test_snap_point.py

```diff
@@ -0,0 +1,33 @@
+import math
+
+import hypothesis
+import hypothesis.strategies as st
+import numpy as np
+import pytest
+
+import habitat_sim
+
+
+@pytest.fixture(scope="function")
+def test_data():
+    pf = habitat_sim.PathFinder()
+    pf.load_nav_mesh(
+        "data/scene_datasets/habitat-test-scenes/skokloster-castle.navmesh"
+    )
+
+    return pf, pf.get_random_navigable_point()
```

Comment:
Will this be deterministic?

Reply:
It's not (seeding the pathfinder is pretty much impossible due to recast using just raw `rand()` and `srand` only kinda working).

I have tried a number of invocations and let hypothesis run out to 100k examples and it couldn't find anything.

Reply:
Maybe add a small comment saying that this isn't deterministic.

Reply:
Done